proposed/internationalisation-meta.md

```diff
@@ -0,0 +1,39 @@
+# Internationalization Meta Document
+
+## 1. Summary
+
+Developers of components that are not coupled to any specific framework often find themselves in need of displaying a message to the user of the web application. However, being as they are not a component of a specific framework - their options for making these messages localisable is limited. In many cases, developers will make no attempt to pass these messages through a localization layer.
+
+To resolve this issue and encourage the localization of libraries, a standard method for translating and transforming messages is necessary.
+
+## 3. Scope
+
+### 3.1. Goals
+
+* Provide a method for an independent component to display a message in a language other than the one in which the component was written in.
+
+### 3.2. Non-Goals
+
+* This PSR does not provide a standard for adding translations of existing messages to a data store (as a language pack may).
```

Comment:
```suggestion
* This PSR does not provide a standard for adding translations of existing messages to a data store (as a language pack may).  That is, this PSR is about denoting that a message is translatable, not providing the specific translations for it.
* This PSR only addresses text present in PHP code.  It is not concerned with the translation of user content stored in a database or similar data store.
```

Reply:
`, not providing the specific translations for it.` is still a bit unclear for me, but I couldn't come up with a better suggestion. Maybe the whole non-goal should be reworded into something like:

```
* This PSR does not provide a standard for storing and managing translatable items and their translations.
  It does not provide a mechanism to collect translatable items.
```

Reply:
I've taken @vdelau's approach, and split the collection of translatable items into another bullet point.